test/org/sagebionetworks/bridge/dynamodb/DynamoUserConsentDaoTest.java

```diff
@@ -28,14 +28,21 @@
     @Before
     public void before() {
         DynamoInitializer.init(DynamoUserConsent2.class);
-        DynamoTestUtil.clearTable(DynamoUserConsent2.class);
+        DynamoInitializer.init(DynamoUserConsent3.class);
+        userConsentDao.deleteConsentRecords(HEALTH_CODE, STUDY_IDENTIFIER);
+        for (int i=1; i < 6; i++) {
+            userConsentDao.deleteConsentRecords(HEALTH_CODE+i, STUDY_IDENTIFIER);
+        }
     }
 
     @After
     public void after() {
-        DynamoTestUtil.clearTable(DynamoUserConsent2.class);
+        userConsentDao.deleteConsentRecords(HEALTH_CODE, STUDY_IDENTIFIER);
+        for (int i=1; i < 6; i++) {
+            userConsentDao.deleteConsentRecords(HEALTH_CODE+i, STUDY_IDENTIFIER);
+        }
     }
-
+    
     @Test
```

Comment:
Can we add a test for error cases (consent2 fails, consent3 fails, both fail (if applicable)) in giveConsent(), just to make sure our error handling logic works the way we expect?